test/unit/test_sap_cli_apirelease.py

```
@@ -341,7 +341,8 @@ def test_set_new_contract(self):
         group = _make_group()
         console = BufferConsole()
         args = _parse(group, 'apistate', 'set', 'c1', 'I_STATISTICALKEYFIGURECAT',
-                       '--state', 'Released', '--cloud-dev', 'No', '--key-user-apps', 'Yes')
+                       '--state', 'Released', '--cloud-dev', 'No', '--key-user-apps', 'Yes',
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)